src/main/java/com/pwn9/filter/bukkit/listener/AbstractBukkitListener.java

```diff
@@ -100,4 +107,14 @@ public void shutdown() {
         }
     }
 
+    boolean checkIfSpam(MessageAuthor minecraftPlayer, String message, Cancellable event){
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/AddstarMC/PwnFilter/pullRequest?prid=6040651)